Start with sorting out 287 (#291)
the-teachingRSE-project/competencies: competencies.md

a story-like description of a fictional RSE career progression in @subsec:examplecareer,
and a list of existing skills and certifications in related fields, in @subsec:existingframeworks.

## The Term Research Software Engineer

> Research Software Engineering considers itself as an interface discipline,
> between traditional Software Engineering, and Research itself.

Reviewer: ```suggestion
Research Software Engineering can be considered as an interface discipline,
linking traditional Software Engineering with Research itself.
```

Reviewer: could cite Anna-Lena in GI-Radar.
https://gi-radar.de/351-research-software/

> This in turn means that there are plethora of different variations of the RSE depending on the 
> particular Research domain they are working in. Therefore the broad notion of Research Software Engineers is better thought of

Reviewer: ```suggestion
Due to this nature there is a plethora of different variations of RSE depending on the 
particular Research domain they are working in. Therefore the broad notion of Research Software Engineers is better thought of
```

as a collection of sub-communities.

> The term of the Research Software Engineer is made more difficult to grasp, since at least least anecdotal evidence suggests,

Reviewer: ```suggestion
The term Research Software Engineer is made more difficult to grasp, since at least  anecdotal evidence suggests,
```

that uk-RSE community has a more technical oriented working definition, de-RSE has a more open working definition that

> allows more people to self-identify as RSEs.

Reviewer: I don't think this is true. My perception is that German institutions struggle more with this term since being called an Engineer in Germany entails particular certification. This exists in the UK as well, but then you are a _chartered engineer_. When I was still in the UK there was some discussion whether RSEs should strive to be recognised in this way as well.

Reviewer 2: I think both are correct. These are two, distinct differences, but both true.

Author: now it is less uk specific.

> Therefore there is a large spectrum across which different definitions of the RSE can be located in.

Reviewer: ```suggestion
RSEs fall therefore somewhere on the spectrum between a researcher at one end and a software engineer at the other.
```

> Common to all of them is, that they  need to be able to work in a Research environment, and therefore
> RSEs potentially need to deal with non-technical complexities that are characteristic for them: organisational, motivational, with respect to the size of projects, independence and heterogeneous goals of stakeholders, boundary conditions for funding and future funding, to name just a few.

Reviewer: ```suggestion
Common to all of them is, that they  need to be able to work in a Research environment. 
RSEs often need to deal with non-technical complexities that are characteristic for Research environments: organisational, motivational, with respect to the size of projects, independence and heterogeneous goals of stakeholders, boundary conditions for funding and future funding, to name just a few.
```

> This variability of how strong the emphasis on the research aspect is, is symbolically represented
> as "R(SE) vs. (RS)E" meaning on the one hand, (SE) in a research environment, which would put it closer to other related fields like automotive software engineering,
> whereas on the other hand we would have engineering practices for Research Software.

Reviewer: ```suggestion
The position on the RSE spectrum can be symbolically represented as "R(SE) vs. (RS)E": the R(SE) is a software engineer in a research environment which would make them akin to a SE in a specialised engineering field such as automotive engineering; the (RS)E is then somebody who applies engineering practices to research software.
```

Reviewer 2: I like this better, but "a software engineer in a research environment" also "applies engineering practices to research software": we need a more specific description for an "RS(E)".

Author: I add some words to amend that.

a story-like description of a fictional RSE career progression in @subsec:examplecareer,
and a list of existing skills and certifications in related fields, in @subsec:existingframeworks.

## The Term Research Software Engineer

Research Software Engineering can be considered an interface discipline,
linking traditional Software Engineering with Research itself [@Lamprecht2024-giradar].
Due to this nature there is a plethora of different variations of RSE depending on the 
particular Research domain they are working in. Therefore the broad notion of Research Software Engineers is better thought of
as a collection of sub-communities.
The term Research Software Engineer is made more difficult to grasp, since an internationally recognised definition

> is still missing. Anecdotal evidence suggests that while there is a broad consensus about the broad notion, that an RSE
> is a person with one leg in research and the other in software development, there seem to be nuances which one is more emphasised.

Reviewer: ```suggestion
is still missing.  While there is a broad consensus about the broad notion, that an RSE
is a person with one leg in research and the other in software development, there arenuances which one is more emphasised.
```

I would just drop the anecdotal evidence and be more definite

> Another dimension is required by the expected level of professionalism. A less strict definition allows more people to self-identify as RSEs.

Reviewer: ```suggestion
There is also the question of what level of professionalism is expected. A less strict definition allows more people to self-identify as RSEs.
```

Maybe also explain that we would prefer to be more inclusive

Author: @mhagdorn I tried to do that, and added a sentence.

a story-like description of a fictional RSE career progression in @subsec:examplecareer,
and a list of existing skills and certifications in related fields, in @subsec:existingframeworks.

## The Term Research Software Engineer

Research Software Engineering can be considered an interface discipline,
linking traditional Software Engineering with Research itself [@Lamprecht2024-giradar].
Due to this nature there is a plethora of different variations of RSE depending on the 
particular Research domain they are working in. Therefore the broad notion of Research Software Engineers is better thought of
as a collection of sub-communities.
The term Research Software Engineer is made more difficult to grasp, since an internationally recognised definition

> is still missing.  While there is a broad consensus about the broad notion, that an RSE

Reviewer: ```suggestion
is still missing.  While there is  consensus about the general notion, that an RSE
```
maybe reduce the number of _broad_s

is a person with one leg in research and the other in software development, there are nuances which one is more emphasised.
There is also the question of what level of professionalism is expected. A less strict definition allows more people to self-identify as RSEs,

> thereby fostering an inclusive community of people working in digital science. This aspect will become a part of an RSEs values later.

Reviewer: ```suggestion
thereby fostering an inclusive community of people working in digital science (see also Section on the values of an RSE).
```

maybe add reference to the value section as well.

Author: thanks! I've added the ref.

a story-like description of a fictional RSE career progression in @subsec:examplecareer,
and a list of existing skills and certifications in related fields, in @subsec:existingframeworks.

## The Term Research Software Engineer

> Research Software Engineering can be considered as an interface discipline,

Reviewer: ```suggestion
Research Software Engineering can be considered an interface discipline,
```

minor grammar fix
